scripts/infrastructure/swagger/merge.sh

```diff
@@ -78,7 +94,8 @@ fi
 # * 4XX codes
 if [[ ${MERGE_AWS} == "1" ]]; then
     cat ${_CLEANED_SWAGGER_FILE} |
-        yq 'del(.. | select(has("4XX")).4XX)' > ${AWS_SWAGGER_FILE}
+        yq 'del(.x-*)' |
+        yq 'del(.. | select(has("4XX")).4XX)' > ${AWS_SWAGGER_FILE} |
```

Comment:
this'll break

Reply:
Are you sure? I can create a valid swagger file from this when I run the merge, unless we want to delete more x- sections that arent on the top level?